### DISCUSSION TITLE
BIDS upload steps if no NWB files

### DISCUSSION THREAD

**Discussion by ucscbrianlee** [dandi/helpdesk#160] [General]
Dear DANDI,

I am doing some preliminary investigation of what non-time-series data might be uploaded into DANDI.  While our group will be doing electrophysiology and optical physiology that is time-series and will be in the NWB format I was curious to learn more about DANDI submissions that would not fit NWB conversions. 

Is there a tutorial about uploading non-NWB files into DANDI in the BIDS approach? 

Here's some areas I looked at for more information on this topic:
- https://www.dandiarchive.org/handbook/30_data_standards/
- - mentions both NWB and BIDS
- https://www.dandiarchive.org/handbook/13_upload/#prerequisites
- - mentions Prerequisites: Convert data to NWB (what about a BIDS submission?)
- https://github.com/dandi/helpdesk/discussions/124
- - mentions 'dandi organize' only works for NWB (also stated in step 5 of 13_upload/#data-uploadmanagement-workflow)

Here's a Dandiset I was looking at that doesn't appear to have any NWB files and is in the BIDS format: 
https://dandiarchive.org/dandiset/000026

While our group will not be submitting that type of data, I was curious to learn more how a submission of that non-NWB approach works, and if there were tutorials/details about such an approach?  

Thank you.

**Comment by kabilar** [comment]
Hi @ucscbrianlee, thanks for reaching out.  Here is a [draft of the instructions](https://github.com/kabilar/handbook/blob/upload/docs/13_upload.md) for uploading a BIDS dataset to DANDI.  This document has been submitted as a pull request in https://github.com/dandi/handbook/pull/123.

**Comment by ucscbrianlee** [comment]
Thank you, Kabilar. I appreciate the response. I haven't tried uploading BIDS before, but reviewing these steps these additions look very helpful!
